users.js

```diff
@@ -46,3 +46,36 @@ const users = [
 	{ "id":45,"name":"Mays Noel","lastly":"00:00 18.04.2017"},
 	{ "id":46,"name":"Solomon Vang","lastly":"10:13 30.11.2017"}
 ];
+
+
+const newUserArr = [];
+const today = new Date();
+function handleUsersData() {
+  users.forEach(user => {
+    const dateParser = user.lastly
+      .slice(6)
+      .replace(/\./gi, "/")
```

Comment:
Zmieniasz kropki na slashe, a potem po nich rozdzielasz stringa, czym różni się to od 

`const dateParser = user.lastly.slice(6).split(".")`

?